app/js/main.js

```diff
@@ -657,14 +766,12 @@ function _videoPlay() {
     // There are no more frames to preview
     if (curPlayFrame >= totalFrames) {
       // We are not looping, stop the playback
-      if (!isLooping) {
-        videoStop();
-      } else {
+      videoStop();
```

Comment:
It looks like here you stop the playback before checking if we are looping, then resume the playback. Was this done for any particular reason?